packages/nocodb-sdk/src/lib/is.ts

```
@@ -94,7 +94,29 @@ export function ncIsString(value: any): value is string {
  * ```
  */
 export function ncIsNumber(value: any): value is number {
-  return typeof value === 'number' && !Number.isNaN(value);
+  return typeof value === 'number' && !Number.isNaN(value);;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)